libsql-server/src/rpc/mod.rs

```diff
@@ -1,6 +1,5 @@
 use std::sync::Arc;
 
-use anyhow::Context;
```

Comment:
All changes in this file seem unrelated. Please post in a separate PR or at least separate commit.